.mci.yml

```diff
@@ -354,6 +354,8 @@ functions:
                   set -o xtrace
 
                   if [ "Windows_NT" == "$OS" ]; then
+                      ENCRYPTION_TESTS_PATH=$(cygpath -m $ENCRYPTION_TESTS_PATH)
```

Comment:
Ah, I think setting the environment variable is needed for a similar reason. In src/mongocxx/test/CMakeLists.txt this overwrites the environment variable test property:

```
if (MONGOCXX_ENABLE_SLOW_TESTS)
  set_tests_properties(driver PROPERTIES
    ENVIRONMENT "MONGOCXX_ENABLE_SLOW_TESTS=1")
endif()
```

While we are cleaning this up, can you update src/mongocxx/test/CMakeLists.txt to append to the environment on the `driver` target?

Reply:
🤦  Yep that's probably it! Appended it instead of overwriting.